l10n_it_account/models/base_language_install.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2024 Sergio Zanchetta (PNLUG APS - Gruppo Odoo)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class BaseLanguageInstall(models.TransientModel):
+    _inherit = "base.language.install"
+
+    def lang_install(self):
+        res = super().lang_install()
+
+        lang_code = self.first_lang_id.code
```

Comment:
Come mai facciamo qualcosa solo quando è il `first_lang_id` e non quando è un altro di `lang_ids`?

Reply:
Ho utilizzato `first_lang_id` perché è il comportamento standard (vedi [qui](https://github.com/odoo/odoo/blob/ef00294e7189359c47638c4a71626f1937395edb/odoo/addons/base/wizard/base_language_install.py#L73))

Esplicito meglio. Questa parte copre il caso in cui il db non viene installato con `--load-language=it_IT` e la lingua italiana viene attivata singolarmente in un momento successivo (vedi [qui](https://github.com/odoo/odoo/blob/ef00294e7189359c47638c4a71626f1937395edb/odoo/addons/base/wizard/base_language_install.py#L42-L50))
La vista collegata è quella che richiama lo [switch](https://github.com/odoo/odoo/blob/ef00294e7189359c47638c4a71626f1937395edb/odoo/addons/base/wizard/base_language_install.py#L72) della lingua.

Ad ogni modo mi sono accorto che non erano gestiti i casi in cui vengono attivate più lingue contemporaneamente, adesso ho apportato le modifiche necessarie.